examples/get_reports_about_intrusion_set.py

```diff
@@ -33,7 +33,7 @@
 reports = opencti_api_client.report.list(
     filters={
         "mode": "and",
-        "filters": [{"key": "contains", "values": [intrusion_set["id"]]}],
+        "filters": [{"key": "objects", "values": [intrusion_set["id"]]}],
```

Comment:
this one was broken since the filter refactoring. objectContains was wrongly renamed to "contains" instead of "objects" (seen with @Archidoit)